src/replacer.rs

```diff
@@ -1,9 +1,25 @@
 use crate::rules::Rules;
 use toml::Value;
+use regex::Regex;
 
 pub fn replace_strings(rules: &Rules, raw: &str) -> String {
     let mut result = raw.trim().to_string();
 
+    // bracket removal
+    for bracket_pair in rules.remove_brackets_list.iter() {
+        if Value::as_array(bracket_pair).unwrap().len() == 2 {
+            let br0 = bracket_pair[0].as_str().unwrap();
+            let br1 = bracket_pair[1].as_str().unwrap();
+            let regex = Regex::new(format!(r#"\{}[^\{}\{}]*\{}"#, br0, br0, br1, br1).as_str()).unwrap();
+            let mut prev = String::from("");
+            while prev != result {
+                prev = result.clone();
+                result = regex.replace_all(prev.as_str(), " ").to_string().replace("  ", " ");
```

Comment:
While the spaces replacement might not cover all languages, I'm fine with this for now. If needed this can still be adjusted in the future.

Reply:
I'm also eager to learn how brackets, punctuation and spaces are handled in 2500+ living languages :)